Tests/AppTests/Util+Snapshotting.swift

```diff
@@ -22,6 +22,10 @@ extension Snapshotting where Value == () -> HTML, Format == String {
 extension Snapshotting where Value == () -> HTML, Format == NSImage {
     public static func image(precision: Float = 1, size: CGSize? = nil, baseURL: URL) -> Snapshotting {
         Current.siteURL = { baseURL.absoluteString }
+
+        // Force light mode
+        NSApplication.shared.appearance = NSAppearance(named: .aqua)
```

Comment:
😮 I thought web views were unaffected from the application's preferences (as seems to be the case from my testing on iOS) -- is macOS different here?

Reply:
Seems to work for me! :)